posts/2025-03-18-github-readmes/index.qmd

```diff
@@ -0,0 +1,234 @@
+---
+title: "README's in GitHub"
+author: "Amrei"
+date: "2025-03-18"
+date-modified: last-modified
+categories: [Github, documentation, self-promotion]
+---
+
+
+# GitHub README's
+
+Every repository in GitHub can have an associated README. The README is often the first thing visitors see. It serves as the **front page** of your project, or your profile, and provides information to visitors and potential contributors. A well-crafted README can increase the accessibility and appeal of your project.
+
+README's can be initialized at the creation of the repository, or added later. They are by default displayed in the code tab of the repository, underneath the files. They are encoded in **Markdown**. 
+
+## profile README
+
+Every personal account can have its own README, which is displayed on top of the profile page. This is a place to share information about you and the work you are doing. 
+
+To initialize the profile README 
+
+- create a repository with a name that matches your GitHub user name
+- make it public
+- add a README
+
+More information on the GitHub documentation pages [GitHub documentation pages](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme).
+
+# Dressing up the README
+
+Regardless of which type of README, there are some things that can be used to make the page more informative and break up the text. Most of the below options were developed for profile README's, but they can also be used on all kinds of repositories.
+
+
+
+## 😎 Add emojis
+
+You can use emojis in the readme to clarify points (and even add them to commits, as I have just learned, use cases are to be determined). 
+
+- 🕰 You can point out deadlines
+
+- [📕](https://www.webfx.com/tools/emoji-cheat-sheet/) Link to documentation
+
+- ✏ Or invite them to contribute
```

Comment:
I can't tell what this emoji is. Initially I thought it was a spill on my screen.